src/common/options.cc

```diff
@@ -1310,6 +1310,23 @@ std::vector<Option> get_global_options() {
     .add_service("mon")
     .set_description("maximum number of OSDMaps to cache in memory"),
 
+    Option("mon_osd_cache_size_min", Option::TYPE_SIZE, Option::LEVEL_ADVANCED)
+    .set_default(128_M)
+    .add_service("mon")
+    .set_description("The minimum amount of bytes to be kept mapped in memory for osd monitor caches."),
```

Comment:
The descriptions look good to me. They can also be added to the docs in doc/rados/configuration/mon-config-ref.rst

Reply:
Done.